projects/netgrif-components/src/lib/panel/task-panel-single/single-task.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, Inject, Optional } from '@angular/core';
+import {
+    AbstractSingleTaskComponent,
+    InjectedTabData,
+    LoggerService,
+    NAE_TAB_DATA
+} from 'netgrif-components-core';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  selector: 'nc-single-task',
+  templateUrl: './single-task.component.html',
+  styleUrls: ['./single-task.component.scss']
+})
+export class SingleTaskComponent extends AbstractSingleTaskComponent {
+
+  constructor(protected _log: LoggerService,
+              @Optional() @Inject(NAE_TAB_DATA) injectedTabData: InjectedTabData,
+              protected route?: ActivatedRoute) {
```

Comment:
change ?
protected route?: ActivatedRoute,
@Optional() @Inject(NAE_TAB_DATA) injectedTabData: InjectedTabData

Reply:
Required member cannot be after optional member.

Reply:
removed optional question mark from route and changed the order